components/skus/renderer/skus_utils.cc

```diff
@@ -23,8 +23,14 @@ bool IsSafeOrigin(const blink::WebSecurityOrigin& origin) {
        {blink::WebSecurityOrigin::Create(
            GURL("https://account.brave.software"))}}};
 
+  const GURL origin_url(origin.ToString().Utf8());
```

Comment:
This specific alert is a false positive because a security origin is created further down.

That said, what's the purpose of removing the port from the origin? Could a comment be added here?